### PR TITLE
R4R:asset tokens sync and asset tokens api 

### PR DESCRIPTION
**Pull request by wenwu-bianjie**
https://github.com/irisnet/explorer/issues/869

### CODE REVIEW

File: backend/service/service_asset.go [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+	BaseService
+}
+
+func (service *AssetService) GetModule() Module {
+	return Validator
+}
+
+func (service *AssetService) UpdateAssetTokens(vs []document.Asset) error {
+	var vMap = make(map[string]document.Asset)
+	for _, v := range vs {
```
Comment (kaifei-bianjie) [inline backend/service/service_asset.go]: unique key should be `asset.id`

File: backend/service/service_asset.go [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+				C:      document.CollectionNmAsset,
+				Id:     v.ID,
+				Remove: true,
+			})
+		}
+	}
+	return document.Asset{}.Batch(txs)
+}
+
+func buildAssetTokens() []document.Asset {
```
Comment (kaifei-bianjie) [inline backend/service/service_asset.go]: there is no need use goroutine to build asset tokens

File: backend/service/service_asset.go (hunk 1 above)
Comment (wenwu-bianjie) [reply backend/service/service_asset.go]: change v.Symbol to v.Id

File: backend/service/service_asset.go (hunk 2 above)
Comment (wenwu-bianjie) [reply backend/service/service_asset.go]: remove gorutine

**Pull request merged**